src/cuda/cufinufft.cu

```diff
@@ -104,54 +103,11 @@ int cufinufft_default_opts(int type, int dim, cufinufft_opts *opts)
 
     opts->gpu_maxbatchsize = 0; // Heuristically set
 
-    switch (dim) {
```

Comment:
This does some sanity checking, which is now gone. This defers it to a later step, which is fine, but we should consider how to best deal with invalid inputs at some point.

Reply:
Right. I'll move it into `makeplan` at some point. Will make an issue.